frames/src/main/res/values/strings.xml

```diff
@@ -27,6 +27,19 @@
     <string name="cko_content_description_image" translatable="true">Image</string>
     <string name="cko_content_description_dynamic_image" translatable="true">Dynamic image</string>
 
+    <!--Billing form screen-->
+    <string name="cko_billing_form_screen_title" translatable="true">Billing address</string>
+    <string name="cko_billing_form_button_save" translatable="true">Save</string>
+    <string name="cko_billing_form_input_field_full_name" translatable="true">Full name</string>
+    <string name="cko_billing_form_input_field_address_line_one" translatable="true">Address line 1</string>
+    <string name="cko_billing_form_input_field_address_line_two" translatable="true">Address line 2</string>
+    <string name="cko_billing_form_input_field_city" translatable="true">City</string>
+    <string name="cko_billing_form_input_field_state" translatable="true">State</string>
+    <string name="cko_billing_form_input_field_postcode" translatable="true">Postcode/zip</string>
+    <string name="cko_billing_form_input_field_phone_title" translatable="true">Phone</string>
+    <string name="cko_billing_form_input_field_phone_subtitle" translatable="true">Phone Number</string>
```

Comment:
Not by design, should be: 
- title "Phone number"
- subtitle "We will only use this to confirm identity if neccessary"

Reply:
Yes. True. However, According to @patrick-hoban-cko, we are required to follow the default UI from the IOS including isoptional fields as well. Please refer [conversation]([@Patrick Hoban](https://checkout.slack.com/team/U02VC518XQU)